_source/logzio_collections/_log-sources/elastic-container-service.md

```diff
@@ -4,47 +4,47 @@ logo:
   logofile: aws-ecs.svg
   orientation: vertical
 open-source:
-  - title: Logz.io AWS ECS Collector
-    github-repo: logzio-aws-ecs
+  - title: docker-collector-logs
+    github-repo: docker-collector-logs
 data-source: Elastic Container Service
 logzio-app-url: https://app.logz.io/#/dashboard/data-sources/ECS
 contributors:
   - imnotashrimp
   - supereli
+  - amosd92
+  - idohalevi
 shipping-tags:
   - aws
 ---
 
-This integration uses Fluentd in a Docker container to forward logs from Amazon Elastic Container Service (ECS) to Logz.io.
+This integration uses Filebeat in a Docker container to forward logs from Amazon Elastic Container Service (ECS) to Logz.io.
 
 ## Setup
 
 To use Logz.io AWS ECS Collector, you'll set environment variables when you run the container.
-The Docker logs directory and docker.sock are mounted to the container, allowing Fluentd to collect the logs and metadata.
+The Docker logs directory and docker.sock are mounted to the container, allowing Filebeat to collect the logs and metadata.
 
 ###### Deploy the AWS ECS collector
 
 1.  Pull the Docker image
 
-    Download the logzio/logzio-aws-ecs image:
+    Download the logzio/docker-collector-logs image.
 
     ```shell
-    docker pull logzio/logzio-aws-ecs
+    docker pull logzio/docker-collector-logs
     ```
 
 2.  Run the Docker image
 
-    For a complete list of options, see the parameters below the code block.👇
+    For a complete list of options, see the parameters below the code block. 👇
 
     ```shell
-    docker run -d --name=logzio-aws-ecs \
+    docker run -d --name=docker-collector-logs \
     --env LOGZIO_TOKEN="<<SHIPPING-TOKEN>>" \
     --env LOGZIO_URL="https://<<LISTENER-HOST>>:8071" \
     -v /var/run/docker.sock:/var/run/docker.sock \
     -v /var/lib/docker/containers:/var/lib/docker/containers \
-    -v /tmp:/tmp \
-    --net="host" \
-    logzio/logzio-aws-ecs
+    logzio/docker-collector-logs
     ```
 
     Parameters
```

Comment:
Do you think we should document all the available params? I see we have only the required one

Reply:
Let's take this in another PR